manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-	"manifest_version": 2,
+	"manifest_version": 3,
 	"name": "The Khan Academy Extension",
 	"short_name": "KA Extension",
 	"author": "Khan Academy Extension Developers",
 	"homepage_url": "https://github.com/ka-extension",
 	"version": "4.7.14",
 	"description": "A Browser Extension for Khan Academy that adds more features to the site",
-	"minimum_chrome_version": "51.0.2704",
+	"minimum_chrome_version": "65.0.3299",
```

Comment:
It might be helpful to know for posterity why this version was chosen

Reply:
Minimum chrome version that supports async generators.  Esbuild isn't able to rewrite those yet.